src/game/client.cpp

```diff
@@ -477,11 +477,14 @@ namespace client
                 items.emplace_back(v >= W_OFFSET && v < W_ITEM ? v : 0);
             }
         }
-        game::player1.loadweap.shrink(0);
-        loopv(items) if(game::player1.loadweap.find(items[i]) < 0)
-        {
-            game::player1.loadweap.add(items[i]);
-            if(game::player1.loadweap.length() >= W_LOADOUT) break;
+        game::player1.loadweap.clear();
+        for (const auto &i : items) {
+            if (std::find(game::player1.loadweap.begin(),
+                game::player1.loadweap.end(), i) == std::end(game::player1.loadweap)) // NOT found
+            {
+                game::player1.loadweap.emplace_back(i);
+                if(game::player1.loadweap.size() >= W_LOADOUT) break;
```

Comment:
Please avoid the use of same-line conditional instructions when touching code. Moving the code to another line (ideally with brackets enclosing the scope) signalizes the branching. I usually also insert a blank line above `if`s etc.

Reply:
Agreed, will fix formatting